src/constants.ts

```diff
@@ -254,3 +254,14 @@ export const ROUNDNESS = {
 /** key containt id of precedeing elemnt id we use in reconciliation during
  * collaboration */
 export const PRECEDING_ELEMENT_KEY = "__precedingElement__";
+
+/**
+ * Unitless line height
+ *
+ * 1.25 aligns with `normal` value in WebKit and Blink. Gecko (FF) uses 1.3.
```

Comment:
Is this also documented anywhere or you did do some test to conclude this?

The standard is `1.2` is what I see documented in [mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/line-height) , [w3c](https://www.w3docs.com/learn-css/line-height.html) where it mentions (`1.1 - 1.2`)

However, I do agree that `1.25` will work well with old versions when it comes to forward compat

Reply:
> Is this also documented anywhere or you did do some test to conclude this?

spec may say one thing (which it doesn't btw, the w3docs has nothing to do with w3c), but the real-world goes brrrr

> However, I do agree that 1.25 will work well with old versions when it comes to forward compat

Not just compat, but because it covers most browser market share

Reply:
To conclude, this was just about `Virgil` which seems to work well with `1.25`. 
Other fonts have slightly diff line heights as expected.
Instead of keeping same for all For `Helvetica` font I would go with 1.15 and for `Cascadia`, I would keep the same as `1.2` for better forward compact. 
let me know what you think ? Also confirm if the above line heights i suggested work for you in windows ? (I am testing on Mac)